apps/edxapp/templates/services/nginx/bc.yml.j2

```diff
@@ -0,0 +1,68 @@
+{%- from "apps/edxapp/templates/services/cms/macros/image.yml.j2" import image_name -%}
+{%- set service="nginx" -%}
+
+apiVersion: "v1"
+kind: "BuildConfig"
+metadata:
+  # FIXME
+  #
+  # As mentioned in the official documentation [1]:
+  #
+  #   Configuration change triggers currently only work when creating a new
+  #   BuildConfig. In a future release, configuration change triggers will also
+  #   be able to launch a build whenever a BuildConfig is updated.
+  #
+  # Hence, we must force BuildConfig to always get created with a unique name.
+  # This is a temporary solution that needs to be improved as soon as OKD
+  # triggers a new build upon BC object update.
+  #
+  # References:
+  #
+  # 1. https://docs.okd.io/latest/dev_guide/builds/triggering_builds.html#config-change-triggers
+  name: "{{ image_name(service, edxapp_nginx_image_tag, edxapp_theme_tag) | replace(':', '-') }}"
+  namespace: "{{ project_name }}"
+  labels:
+    app: "edxapp"
+    service: "{{ service_variant }}"
+    version: "{{ edxapp_image_tag }}"
+    theme: "{{ edxapp_theme_tag | default("none") }}"
+spec:
+  strategy:
+    type: Docker
+  source:
+    {% if edxapp_theme_url is defined and edxapp_theme_url -%}
+    git:
+      uri: "{{ edxapp_theme_url }}"
+      ref: "{{ edxapp_theme_tag | default("master") }}"
+    {% if edxapp_theme_is_private -%}
+    sourceSecret:
+      name: "{{ edxapp_theme_git_private_key_secret_name }}"
+    {% endif -%}
+    {% endif -%}
+    dockerfile: |-
+      {% if edxapp_theme_url is defined and edxapp_theme_url -%}
+      FROM {{ edxapp_image_name }}:{{ edxapp_image_tag }} as edxapp
+      USER 0
+      RUN apt-get update && \
+        ( which npm || apt-get install -y npm ) && \
+        apt-get install -y rdfind && \
+        rm -rf /var/lib/apt/lists/*
+      COPY . /edx/app/edxapp/edx-platform/themes/custom-theme
+      RUN NO_PREREQ_INSTALL=1 paver update_assets --settings={{ edxapp_build_settings }} --skip-collect
+      # Replace duplicated file by a symlink to decrease the overall size of the
```

Comment:
And we also should add frontend messages translation support, see: https://github.com/openfun/arnold/blob/v4.7.0/apps/edxapp/templates/services/lms/job_04_collectstatic.yml.j2#L42